src/Integration/Binding/BindingProcessImpl.cs

```diff
@@ -97,52 +97,78 @@ internal class BindingProcessImpl : IBindingProcess
 
         public async Task<bool> DownloadQualityProfileAsync(IProgress<FixedStepsProgress> progress, CancellationToken cancellationToken)
         {
-            var languageList = this.GetBindingLanguages();
+            var languageList = GetBindingLanguages();
 
             var languageCount = languageList.Count();
             int currentLanguage = 0;
-            progress?.Report(new FixedStepsProgress(Strings.DownloadingQualityProfileProgressMessage, currentLanguage, languageCount));
```

Comment:
I removed this initial progress message. It only appeared briefly, until the first QP was downloaded. The new messaging is hopefully more useful.